app/views/moderator_applications/_form.html.slim

```diff
@@ -0,0 +1,55 @@
+section.card
+  header.card-header
+    strong Moderator Application Form
```

Comment:
`strong` is not a semantic tag. This should be an `hX` tag. Just because I'd have to write a bit of CSS to make it work is no excuse to be lazy.

Reply:
I'd really like it to be an `h1` tag universally, but unfortunately the proposed HTML5 new document outline algorithm was rejected and that's no longer a viable option. Oh well. Technically hardcoding it as an `h2` would work because it's only used in an `h2` document outline context anyway, but as-is that would break the formatting, which would require writing more CSS, like I mentioned in my previous comment.

Reply:
"Just because i'd have to write ab it of CSS to make it work is no excuse to be lazy"
didn't you write this bit of code? are you being harsh on yourself or trying to bring up a certain point?

Reply:
I did. I was being harsh on myself.